Libraries/D3D12RaytracingFallback/src/ConstructAABBBindings.h

```diff
@@ -17,14 +17,16 @@
 struct InputConstants
 {
     uint NumberOfElements;
+    uint PrepareUpdate;
```

Comment:
HLSL is annoying and will pad both of these out to 2 full DWORDs (4 bytes each). Can you pack these into a single uint and use bit masks to query which is set?